bug - parseEmail does not remove "RE:" prefix in subject (#35884)
demisto/content: Packs/EmailCommunication/ReleaseNotes/2_0_31.md

#### Scripts

##### SendEmailReply

> - Fixed an issue where the parsing of the email was not handled correctly when the subject had a "RE:" prefix.

Reviewer: ```suggestion
- Fixed an issue where the "RE:" prefix was not removed correctly from the email subject.
```
